scripts/utils/black_check.py

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env python
-"""Run the black check on python 3, do nothing on python 2."""
```

Comment:
Also, understood that this wrapper existed in the original commit, I missed it when reviewing the 1000s of files changed. I would've raised the same objection wrt my other comment about wrappers.